config/crd/bases/addons.managed.openshift.io_addonimagesets.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.7.0
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
controller-gen was bumped to 0.7.0 in main, probably need a rebase? wonder why Github is not reporting conflicts... :thinking:

Reply:
Ohh nice catch! So, I just found out that the `make generate` downloads controller-gen binary in `.cache/bin/` directory. But if .cache is present, then, controller-gen doesn't explicitly download the required controller-gen binary but straight away uses the one present in .cache.

And long in the past, maybe the first time I ran `make generate` in my project directory locally, .cache got setup with controller-gen v0.4.1 binary and since then, whenever I run `make generate`, straight away that 0.4.1 binary gets used instead of actually downloading and using the one required in Makefile.

Well as Phil Karlton said, "there are only two hard things in computer science cache invalidation and naming things"

Reply:
Fixed!